radis/api/kuruczapi.py

```diff
@@ -0,0 +1,451 @@
+"""
+The AdBKurucz class below is inspired by an Exojax class developed by Hiroyuki Tako ISHIKAWA.
+It allows loading data from the Kurucz database and performing several calculations on it.
+
+Author: Racim Menasria
+Date: June 2023
+"""
+
+
+import io
+import pkgutil
+from contextlib import closing
+from io import BytesIO
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import periodictable
+import requests
+from tqdm import tqdm
+
+
+class AdBKurucz:
```

Comment:
MAJOR COMMENT. 
You should be inheriting from Radis API main Databases classes ; if you're simply rewriting another parser on the side it won't improve our goal towards a CommonAPI